public/kubernetes-guides/advanced-guides/dynamic-resource-allocation.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Dynamic Resource Allocation"
+description: "Request and share node level resources among Kubernetes pods."
+---
+
+[Kubernetes Dynamic Resource Allocation](https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA) is a new way of sharing host level resources into pods.
+These resources include GPU devices, high performance networking, and other hardware that a workload may need access to.
+
+DRA is enabled by default as a beta feature in Kubernetes 1.34 and can also be enabled in Kubernetes 1.33.
+DRA replaces [devices plugins](./device-plugins.mdx) for accesing resources from workloads.
+
+## Enable DRA in the cluster
+
+Make sure the cluster has DRA enabled (Kubernetes 1.34+) or uses the following patch to enable the feature. This patch can be applied to all nodes in the cluster via `talosctl` or via Omni.
+
+```yaml
+machine:
+  kubelet:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+cluster:
+  apiServer:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+  controllerManager:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+  scheduler:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+```
+
+You should have at least one node in the cluster with NVIDIA hardware and configured via the [NVIDIA system extension and patch](../../talos/v1.11/configure-your-talos-cluster/hardware-and-drivers/nvidia-gpu-proprietary).
+
+## Deploy the NVIDIA DRA plugin
+
```

Comment:
nit, usually for visual hierachy we always recommend that after a title or subtitle, you'd add a sentence or so instead of following with another title. something that gives us an idea of what this section is about.